### PR TITLE
Change application name and meta data

### PR DESCRIPTION
**Pull request by sebastien-comeau**
### Description

List of proposed changes:

- Make the page title to reflect of page h1 or application
- Change the application name in English `Passport application status checker` and French `Outil de vérification de l'état de la demande de passeport`
- Change `statut` to `état` in French content

### What to test for/How to test

Ensure the application title is correct in both English and French.

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -18,7 +18,8 @@ const Index = () => {
         <div className="m-auto w-[300px] md:w-[400px] lg:w-[500px] bg-gray-lighter">
           <div className="p-8">
             <h1 className="sr-only">
-              service.canada.ca-passport-status-checker
+              Passport application status checker | Outil de vérification de
+              l&apos;état de la demande de passeport
```
Comment (gregory-j-baker) [inline pages/index.tsx:22]: `&apos;` should be changed to `'` to be consistent with the rest of the application.

**Review by gregory-j-baker — CHANGES_REQUESTED** [review]

File: pages/index.tsx (hunk 1 above)
Comment (sebastien-comeau) [reply pages/index.tsx:22]: We are actually using `&#39;` in other pages. Eslint will complains if we don't escape `'` in the html.
image

Comment (sebastien-comeau) [reply pages/index.tsx:22]: I've updated that file with `&#39;`

Comment (sebastien-comeau) [reply pages/index.tsx]: Because it's a bilingual page and not translated.

**Review by gregory-j-baker — APPROVED** [review]

**Pull request merged**